DashAI/back/models/scikit_learn/sklearn_like_model.py

```diff
@@ -22,41 +21,24 @@ def load(filename: str) -> None:
 
     # --- Methods for process the data for sklearn models ---
 
-    def format_data(self, dataset: DashAIDataset) -> Tuple[pd.DataFrame, pd.Series]:
-        """Load and prepare the dataset into dataframes to use in Sklearn Models.
-
-        Parameters
-        ----------
-        dataset : DashAIDataset
-            Dataset to format
-
-        Returns
-        -------
-        Dataframe
-            Dataframe with the data to use in experiments.
-        """
-        data_in_pandas = dataset.to_pandas()
-        x = data_in_pandas.loc[:, dataset.inputs_columns]
-        y = data_in_pandas[dataset.outputs_columns]
-
-        return x, y
-
-    def fit(self, dataset: DashAIDataset) -> Type["SklearnLikeModel"]:
+    def fit(self, x: DashAIDataset, y: DashAIDataset) -> Type["SklearnLikeModel"]:
         """Fit the estimator.
 
         Parameters
         ----------
-        dataset : DashAIDataset
-            The training dataset.
+        x : pd.DataFrame
+            Dataframe with the input data.
+        y : pd.DataFrame
+            Dataframe with the output data.
```

Comment:
Why not to use `X_train` and `y_train` here , and in `X_pred` in the prediction function?